<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Extension_Form</class>
 <widget class="QWidget" name="Extension_Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>290</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Extension</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Add New Work</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Add_BT">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>100</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Close_BT">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>250</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="FindFile_BT">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>40</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Find Work File</string>
   </property>
  </widget>
  <widget class="QPushButton" name="FindWeb_BT">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>70</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Find in Web</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Extension</string>
   </property>
  </widget>
  <widget class="QListWidget" name="NewWork_listWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>281</width>
     <height>211</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="Delete_BT">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>220</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Close_BT</sender>
   <signal>clicked()</signal>
   <receiver>Extension_Form</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>336</x>
     <y>264</y>
    </hint>
    <hint type="destinationlabel">
     <x>344</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
